### DISCUSSION TITLE
Migration for deleted models

### DISCUSSION THREAD

**Discussion by amshehzad** [spatie/laravel-deleted-models#17] [Ideas]
Hi,
Thank you for this great package!
I have a question regarding the working of the package. Currently, we copy all attributes to the delete_model table. but what if we rename a column after deleting some records? the restoration won't work correctly. I think that there should be an out-of-the-box way to run migrations for deleted_models which will rename the columns in the deleted_models data as well.
I'm not sure if it's the best way of doing it, but I would be looking forward to suggestions on this

**Comment by freekmurze** [comment]
Thanks for the kind words

Restoring records and syncing column names in deleted data is out of scope for this package, so handle that in your own project (or fork our package).
